scripts/run_integration_tests.sh

```diff
@@ -36,6 +36,12 @@ cd $integration_tests_dir
 for ((i = 0 ; i < ${#SERVERLESS_CONFIGS[@]} ; i++)); do
     echo "Running 'sls package' with ${SERVERLESS_CONFIGS[i]}"
     serverless package --config ${SERVERLESS_CONFIGS[i]}
+    # Normalize unstable values, perl substitution command: perl -p -i -e 's/(PATTERN TO FIND)/PATTERN TO SUBSTITUTE/g'
```

Comment:
Nit: I wouldn't document how perl works here -- engineers can Google that if they want to look it up